docs/getting-started.adoc

```diff
@@ -1,4 +1,94 @@
-== Getting started
+== Getting Started with AeroGear Services JavaScript SDK
 
-SDK is under construction. 
-No SDK available at the moment.
+This guide provides general information for creating applications that utilize the AeroGear Services JavaScript SDK.
+
+== Prerequisites for Apache Cordova
+
+In order to develop a Cordova application utilizing the SDK you will need the following prerequisites installed:
```

Comment:
I would assume the user has already a Cordova app, instead of trying to explain the whole process. We offer a service for Cordova apps so I would say `In order to include/use this SDK in your Apache Cordova application...` and omit the `Creating an Apache Cordova app` that doesn't say much.

Reply:
:+1:  I'll go ahead and remove the 'creating an app' section, but for consistency with what's in the android sdk (which lists downloading android studio, etc as prerequisites), I was thinking of actually keeping this part.